Content.Server/StationEvents/Metric/Components/FoodMetricComponent.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Content.Server.StationEvents.Events;
+using Content.Shared.FixedPoint;
+
+namespace Content.Server.StationEvents.Metric.Components;
+
+[RegisterComponent, Access(typeof(FoodMetric))]
+public sealed class FoodMetricComponent : Component
+{
+    [DataField("thirstScore"), ViewVariables(VVAccess.ReadWrite)]
+    public readonly FixedPoint2 ThirstScore = 2.0f;
+
+    [DataField("parchedScore"), ViewVariables(VVAccess.ReadWrite)]
+    public readonly FixedPoint2 ParchedScore = 5.0f;
+
+    [DataField("peckishScore"), ViewVariables(VVAccess.ReadWrite)]
+    public readonly FixedPoint2 PeckishScore = 2.0f;
+
+    [DataField("starvingScore"), ViewVariables(VVAccess.ReadWrite)]
+    public readonly FixedPoint2 StarvingScore = 5.0f;
+}
```

Comment:
This should just be a dictionary not 5 different fields.

Reply:
Really? Then we get back into magic strings (in the dictionary)